app/javascript/App/components/NewsfeedPage/NewsListHeader.js

```diff
@@ -1,30 +1,64 @@
 import React, { Fragment } from 'react'
 import Icon from '../Icon'
 import SectionHeader from './SectionHeader'
+import SectionHeaderTight from './SectionHeaderTight'
 import SearchCoins from './SearchCoins'
 import FilterPanel from '../FilterPanel'
 import FilterTags from '../FilterTags'
+import bulbIcon from '../../images/bulbIcon.svg'
 
 const NewsListHeader = (props) => {
-  const { enableUI, currentUI, coins, feedSources, activeFilters } = props
+  const { enableUI, currentUI, coins, feedSources, activeFilters, activeEntity, newsfeedTips } = props
   const toggleFilters = () =>
     enableUI('filterPanel', { fullScreen: window.isMobile })
+
+  const btnStyle = {
+    padding: '18px',
+    borderRadius: 0
+  }
+
   return (
     <Fragment>
-      <SectionHeader>
-        <div className="flex-auto flex items-center">
+      {window.isMobile && (
+          <SectionHeaderTight>
```

Comment:
Seems like the indentation is a bit weird

Reply:
thanks for pointing this out!